src/components/Lenkepanel/LenkepanelWrapper.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Element, Undertekst, Undertittel } from 'nav-frontend-typografi';
+import { LenkepanelBase } from 'nav-frontend-lenkepanel';
+
+import './LenkepanelWrapper.less';
+import { Periode, Arbeidsgiver, Sykmelding } from '../../types/sykmeldingTypes';
+import dayjs from 'dayjs';
+
+interface LenkepanelProps {
+    lenke: string;
+    tittel: string;
+    sykmeldinger: Sykmelding | Sykmelding[];
+    tekst: string;
+    svg: string;
+}
+
+const beregnPeriodelengde = (fom: Date, tom: Date): number => dayjs(tom).diff(dayjs(fom), 'day');
+const tilLesbarPeriodeMedGraderingOgArbeidsgiver = (
+    periode: Periode,
+    arbeidsgiver: Arbeidsgiver,
+    periodelengde: number,
+): string => {
+    const gradering = periode.gradert && periode.gradert.grad ? periode.gradert.grad : 100;
+    const arbeidsgiverNavn = arbeidsgiver.navn ? arbeidsgiver.navn : 'Ukjent arbeidsgiver';
+
+    return arbeidsgiverNavn + ' • ' + gradering.toString() + '% i ' + periodelengde + ' dager';
+};
+
+const LenkepanelWrapper = ({ lenke, tittel, sykmeldinger, tekst, svg }: LenkepanelProps) => {
+    return (
+        <LenkepanelBase border href="" linkCreator={linkProps => <Link {...linkProps} to={lenke} />}>
+            <div className="lenkepanelwrapper-container">
+                <img src={svg} width={60} className="lenkepanelwrapper-bilde" alt="Lenkepanelillustrasjon" />
+                <div className="lenkepanelwrapper-tekst">
+                    <Undertittel className="lenkepanel__heading">{tittel}</Undertittel>
+                    {sykmeldinger instanceof Sykmelding
```

Comment:
Tenker dette burde skilles ut til en egen funksjon (feks "hentPerioder"), og i tillegg refaktoreres slik at ikke samme kode skrives to ganger.

Feks hent ut perioder fra sykmelding/sykmeldinger først, og så mappe over periodene.